### ISSUE TITLE
Some icons are not compatible with Retina display, and an alignment issue

### ISSUE THREAD

**Issue by starkshaw** [rsms/fb-mac-messenger#298]
<img width="790" alt="screen shot 2016-05-18 at 19 36 58" src="https://cloud.githubusercontent.com/assets/9618357/15370715/129d0868-1d30-11e6-8b83-241938f314d8.png">

These 2 icons are blurred.

<img width="406" alt="screen shot 2016-05-18 at 19 37 09" src="https://cloud.githubusercontent.com/assets/9618357/15370716/12bf2740-1d30-11e6-9c20-77430d1aa5af.png">

OS X 10.11.5

**Comment by drasbeck** [comment]
Here's an overview of which icons on the main page that lacks retina support, captured on a MacBook Pro (Retina, Mid 2012), OS X El Capitan 10.11.5 (15F34).

![messenger](https://cloud.githubusercontent.com/assets/8578706/15645095/e2e82f96-2656-11e6-8f96-1b400b3777dd.jpg)

**Comment by starkshaw** [comment]
@drasbeck Yes I have the same. But if you compare the app with messenger.com You'll notice these icons are blurred. 😃

**Comment by drasbeck** [comment]
Thought as much, I don't use messenger.com but seeing as this is a wrapper for that page, it makes sense. Does this mean it's not fixable?

**Comment by stillmoms** [comment]
All of these icons are Retina-rendered when I go to messenger.com (along with Facebook's smileys) in Safari on OS X 10.11, but not in the app. Is there something in the app that's not being set correctly to pull the proper assets from the web app?

**Comment by stillmoms** [comment]
I see everything now loading in Retina quality on my Retina 5K iMac, but on my 12" MacBook (Early 2016), still seem to have blurry assets. Reloading from server doesn't help; any kind of cache I can clear?

**Comment by yesthisisjoe** [comment]
I just downloaded the app for the first time and the same icons @drasbeck referenced are still blurry for me. On macOS 10.12 (16A304a)
